requirements/reproducible-linux.txt

```diff
@@ -296,9 +296,9 @@ text-unidecode==1.3 \
     --hash=sha256:1311f10e8b895935241623731c2ba64f4c455287888b18189350b67134a822e8 \
     --hash=sha256:bad6603bb14d279193107714b288be206cac565dfa49aa5b105294dd5c4aab93
     # via python-slugify
-tomlkit==0.7.0 \
-    --hash=sha256:6babbd33b17d5c9691896b0e68159215a9387ebfa938aa3ac42f4a4beeb2b831 \
-    --hash=sha256:ac57f29693fab3e309ea789252fcce3061e19110085aa31af5446ca749325618
+tomlkit==0.7.2 \
```

Comment:
We need to make sure we update our THIRD-PARTY-LICENSES file: https://github.com/aws/aws-sam-cli/blob/develop/installer/assets/THIRD-PARTY-LICENSES#L2490